### DISCUSSION TITLE
[Wiki/Tips] `dmulholl/vimv` no longer has `--git` option in current version

### DISCUSSION THREAD

**Discussion by xfzv** [gokcehan/lf#1365] [Ideas]
From https://github.com/gokcehan/lf/wiki/Tips#bulk-rename-multiple-files

```
cmd bulkrename ${{
    vimv --git -- $(basename -a -- $fx)

    lf -remote "send $id load"
    lf -remote "send $id unselect"
}}

map R bulkrename
```

This will fail because current `vimv` version ([3.0.0](https://github.com/dmulholl/vimv/releases/tag/3.0.0) as I'm writing this) no longer has `--git` option:

```console
❯ vimv --git *
Error: --git is not a recognised flag or option name.
```

It's no longer documented on [author's website](https://www.dmulholl.com/dev/vimv.html) nor in `vimv --help`:

```console
Arguments:
  [files]                   List of files to rename.

Options:
  -e, --editor <name>       Specify the editor to use. Overrides $EDITOR.

Flags:
  -f, --force               Allow overwriting existing files.
  -h, --help                Print this help text and exit.
  -q, --quiet               Quiet mode -- only report errors.
  -s, --stdin               Read the list of input files from standard input.
  -v, --version             Print the version number and exit.
```

Please consider removing `--git` from `bulkrename` command as it solves the issue.

**Comment by joelim-work** [comment]
My understanding is that the wiki is open to the community and can be edited by anyone, so feel free to make such changes.

Otherwise if you don't have permission then let me know and I can edit it for you.

**Comment by xfzv** [comment]
My bad, I thought the wiki could only be edited by contributors. I've just edited it.
